crt_portal/cts_forms/models.py

```diff
@@ -1172,6 +1172,8 @@ def available_report_fields(self, report: Optional[Report]):
         return Context({
             'record_locator': report.public_id,
             'addressee': report.addressee,
+            'complainant_name': report.contact_first_name + " " + report.contact_last_name,
```

Comment:
```suggestion
            'complainant_name': f'{report.contact_first_name} {report.contact_last_name}',
```

Prefer a format string. This will fail if either variable is null / `None`:

```py
Python 3.13.0 (main, Nov 26 2024, 10:35:00) [Clang 14.0.3 (clang-1403.0.22.14.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> None + " " + None
Traceback (most recent call last):
  File "<python-input-0>", line 1, in <module>
    None + " " + None
    ~~~~~^~~~~
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
>>>
```